lib/serve-asset-middleware.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var serveAsset = require('./utils/serve-asset');
+var setResponseHeaders = require('./utils/response-header').setResponseHeaders;
+
+module.exports = function serveAssetsMiddleware() {
```

Comment:
I was just following the format of other middlewares for consistency. Fixed to just not have the closure 😄